portal/static/js/main.js

```diff
@@ -5970,7 +2452,7 @@ var tnthDates = {
      * changeFormat("2016-04-29",true) returns "29/04/2016"
      * changeFormat("2016-04-29",true,true) returns "29/04/2016"
      ***/
-    "changeFormat": function(currentDate,reverse,shorten) {
+    "changeFormat": function(currentDate, reverse, shorten) {
```

Comment:
This might be simpler if you break the parsing and the formatting into 2 different functions